node/full.go

```diff
@@ -136,9 +139,9 @@ func (qm *QitmeerFull) RegisterNotifyMgr() error {
 	return nil
 }
 
-func (qm *QitmeerFull) RegisterAccountService(cfg *config.Config) error {
+func (qm *QitmeerFull) RegisterAccountService(cfg *config.Config, events *event.Feed) error {
 	// account manager
-	acctmgr, err := acct.New(qm.GetBlockChain(), cfg)
+	acctmgr, err := acct.New(qm.GetBlockChain(), cfg, events)
```

Comment:
same reason here, problem is remaining, The API should not changed in this way. 
Please reconsider the design. when the service component is depends on each others.
when it should be created, and how to be constructed into a working node.